src/app/src/components/FilterSidebarSearchTab.jsx

```diff
@@ -140,10 +140,10 @@ function FilterSidebarSearchTab({
             <ol>
                 <li style={styles.tooltipLineItem}>
                     You can search for all the facilities of multiple
-                    contributors. this means that the results would show all of
-                    the facilities contributed to the oar by, for example, brac
-                    university or clarks. some facilities might have been
-                    contributed by brac university but not by clarks, or
+                    contributors. This means that the results would show all of
+                    the facilities contributed to the OAR by, for example, BRAC
+                    University or Clarks. Some facilities might have been
+                    contributed by BRAC university but not by Clarks, or
```

Comment:
University should be capitalized here

Reply:
Thanks. Pushed fixup 424bff5 to correct this.